scripts/setupv2.sh

```diff
@@ -87,7 +103,23 @@ while true; do
     break
     ;;
 
-  *) echo "Please enter a number from 1 to 4." ;;
+  5)
+    echo "> Citadel"
+    echo
+    isDocker=1
+    dockerNetwork="a-docker-tunnelsats"
+    dockerScriptPrefix="citadel"
+    dockerMainDir=$(find / -maxdepth 5 \( -path /mnt -prune -o -path /sd-root -prune \) -o -type f -name "karen" -print 2>/dev/null | awk -F '/karen' '{print $1}')
```

Comment:
`dockerMainDir`: find command also finds temporary update paths of Citadel `tmp/citadel-update/`
another solution to be discussed here
```suggestion
    dockerMainDir=$(find / -maxdepth 5 \( -path /mnt -prune -o -path /sd-root -prune -o -path /tmp -prune \) -o -type f -name "karen" -print 2>/dev/null | awk -F '/karen' '{print $1}')
```
untested suggestion